source/jormungandr/jormungandr/instance.py

```diff
@@ -86,6 +86,7 @@ def __init__(self, context, name, zmq_socket, realtime_proxies_configuration=[])
                                                 reset_timeout=app.config['CIRCUIT_BREAKER_INSTANCE_TIMEOUT_S'])
         self.georef = georef.Kraken(self)
         self.planner = planner.Kraken(self)
+        self.street_network = street_network.StreetNetwork.get_street_network(self, street_network_configuration)
```

Comment:
minor: I prefer naming it self.street_network_service/self.street_network_worker